DEVELOPMENT.rst

```diff
@@ -107,32 +107,91 @@ All other commands are the same as for GNU/Linux and Mac OS X.
 a. Running Angular.js test cases
 ''''''''''''''''''''''''''''''''
 
+Run them by starting karma::
+
     $ node_modules/.bin/karma start tests/karma/karma.conf.js
 
 
 Installation OpenSlides in big mode
 ===================================
 
-1. Install PostgreSQL und redis:
+1. Install PostgreSQL and redis
+-------------------------------
+
+    Just apt-get these packets::
+
+    $ sudo apt-get install postgresql redis-server libpq-dev --yes
+
+2. Configure postgresql
```

Comment:
Just installed PostgreSQL via apt-get and then added DB-user with password and DB with the commands given here: https://wiki.ubuntuusers.de/PostgreSQL/#Administration.